### ISSUE TITLE
Arrange an accessibility best practice session for VSA Global UX

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#924]
* Best practices
* Focus management
* HTML structure

As a developer, I want an accessibility workshop so that I understand the best practices when developing with accessibility in mind.

**Comment by aeltarhoni** [comment]
@1Copenut Is this for the Global UX Team?

**Comment by 1Copenut** [comment]
@aeltarhoni Yes, this should be assigned to the PM for the VSA Global team and we can schedule a 508 meeting to discuss best practices and expectations around audits and timing.

**Comment by ncksllvn** [comment]
Thank you @1Copenut! Assigning @Karak888 :smile:

**Comment by 1Copenut** [comment]
@Karak888 I've got a demo ready for talking 508 with the VSA Global team. LMK if there's a time that works well for your team. I recommend 30-60 minutes. Thank you.

**Comment by Karak888** [comment]
@1Copenut I will reach out to you about getting this scheduled via DM, thanks!

**Comment by 1Copenut** [comment]
Reschedule for Friday, October 4th.

**Comment by Karak888** [comment]
closing this piece out as we are scheduled for Friday as Trevor has indicated above!
